src/home_robot/home_robot/utils/point_cloud.py

```diff
@@ -53,30 +46,114 @@ def show_pcd(
     orig: np.ndarray = None,
     R: np.ndarray = None,
     save: str = None,
-    grasps: list = [],
+    grasps: list = None,
 ):
+    geoms = create_visualization_geometries(pcd=pcd, orig=orig, R=R, grasps=grasps)
+    o3d.visualization.draw_geometries(geoms)
+
+    if save is not None:
+        save_geometries_as_image(geoms, output_path=save)
+
+
+def create_visualization_geometries(pcd=None, xyz=None, rgb=None, orig=None, R=None, size=1.0,
+                     arrow_pos=None, arrow_size=1.0, arrow_R=None, arrow_color=None,
+                     sphere_pos=None, sphere_size=1.0, sphere_color=None, grasps=None):
```

Comment:
we're trying to add default types + a short comment - if that's not too much trouble